src/Compilers/Core/Portable/CommandLine/AnalyzerConfig.cs

```diff
@@ -106,7 +106,7 @@ public sealed partial class AnalyzerConfig
         /// Parses an editor config file text located at the given path. No parsing
         /// errors are reported. If any line contains a parse error, it is dropped.
         /// </summary>
-        public static AnalyzerConfig Parse(string text, string pathToFile)
+        public static AnalyzerConfig Parse(string text, string? pathToFile)
```

Comment:
In annotating `CommonCompiler` and looking at other call sites it's clear that `null` is an allowed value here.